phabricator-user.rst

```diff
@@ -386,21 +380,16 @@ the main components:
   extensions, we may move them to
   `bugzilla.mozilla.org :: Extensions: PhabBugz
   <https://bugzilla.mozilla.org/enter_bug.cgi?product=bugzilla.mozilla.org&component=Extensions%3A%20PhabBugz>`_
-  depending on where the problem exists in our code.  Also note that, as
-  discussed in :ref:`bmo-integration`, we are strictly limiting
-  customizations to our instance.  We may, however, work with upstream
-  in fixing important issues.
+  depending on where the problem exists in our code.  Also note that, as discussed in ref:`bmo-integration`, we are strictly limiting customizations to our instance.  We may, however, work with upstream in fixing important issues.
```

Comment:
Broken `:refs:`, and a change that mostly folds multiple lines unnecessarily; drop this change.